services/management/annotation_test.go

```diff
@@ -0,0 +1,267 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package management
+
+import (
+	"context"
+	"testing"
+
+	"github.com/AlekSi/pointer"
+	"github.com/google/uuid"
+	"github.com/percona/pmm/api/managementpb"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"gopkg.in/reform.v1"
+	"gopkg.in/reform.v1/dialects/postgresql"
+
+	"github.com/percona/pmm-managed/models"
+	"github.com/percona/pmm-managed/utils/logger"
+	"github.com/percona/pmm-managed/utils/testdb"
+	"github.com/percona/pmm-managed/utils/tests"
+)
+
+func TestAnnotations(t *testing.T) {
+	setup := func(t *testing.T) (ctx context.Context, db *reform.DB, teardown func(t *testing.T)) {
+		t.Helper()
+
+		ctx = logger.Set(context.Background(), t.Name())
+		uuid.SetRand(new(tests.IDReader))
+
+		sqlDB := testdb.Open(t, models.SetupFixtures, nil)
```

Comment:
Let's skip fixtures since we don't use them.

Reply:
OK.

Reply:
@BupycHuk With models.SkipFixtures I getting error: rpc error: code = NotFound desc = Node with ID "pmm-server" not found. I using this node for tests where is only service to save some lines of creating another.

Reply:
oh, yeah, missed that.